addons/api/tests/unit/serializers/target-test.js

```diff
@@ -36,6 +36,7 @@ module('Unit | Serializer | target', function (hooks) {
       address: '0.0.0.0',
       enable_session_recording: false,
       storage_bucket_id: null,
+      with_aliases: [{ value: 'a', scope_id: 'global' }],
```

Comment:
If the serializer adds `scope_id` within `with_aliases` shouldn't attempt to create the record without passing `scope_id`? so we validate the serializer is correctly adding the `scope_id: 'global'`?

Reply:
Ooh that is a good point, I'll update accordingly